management/grant_test.go

```diff
@@ -7,10 +7,10 @@ import (
 )
 
 func TestGrantManager_List(t *testing.T) {
-	setupHTTPRecordings(t)
+	configureHTTPTestRecordings(t)
 
-	grantList, err := m.Grant.List()
+	grantList, err := api.Grant.List()
 	assert.NoError(t, err)
 	assert.IsType(t, &GrantList{}, grantList)
-	assert.Len(t, grantList.Grants, 0)
+	assert.NotNil(t, grantList.Grants)
```

Comment:
We should have just checked that we get some grants back not that they're actually 0, as the test tenant could have users with assigned grants.